helper.mac.fish

```diff
@@ -77,22 +81,98 @@ function findRequiredMinMacOS
 end
 
 function opensslVersion
-  set -l oversion $argv[1]
+  set -gx OPENSSL_VERSION $argv[1]
+end
 
-  if test "$oversion" = ""
-    set -e OPENSSL_VERSION
-    return 0
+function downloadOpenSSL
+  if test -z $OPENSSL_VERSION
+    echo "Need OPENSSL_VERSION global variable!"
```

Comment:
Please improve this helper output.
What should be the content of the Variable? What is the "default"?

Reply:
shouldn't this use the contents of the `VERSIONS` file in the source: 
```
OPENSSL_MACOS   "1.1.1l"
```

Reply:
Fixed.